go/tools/gopackagesdriver/bazel_json_builder.go

```diff
@@ -93,12 +88,7 @@ func (b *BazelJSONBuilder) getKind() string {
 }
 
 func (b *BazelJSONBuilder) localQuery(request string) string {
-	request = path.Clean(request)
-	if filepath.IsAbs(request) {
-		if relPath, err := filepath.Rel(workspaceRoot, request); err == nil {
-			request = relPath
-		}
-	}
+	request = path.Clean(b.adjustToRelativePathIfPossible(request))
```

Comment:
`filepath.Rel` always returns a clean path, so this wouldn't be necessary if we figure out the error case.

Reply:
also removing the clean given it is very unlikely for us to reach the error scenario here